packages/icons/svgo-jsx.config.ts

```diff
@@ -0,0 +1,53 @@
+import * as path from "path";
+
+const template = ({
+  sourceFile,
+  componentName,
+  jsx,
+}) => `// Generated from ${sourceFile}
+
+import { forwardRef } from "react";
+import { IconProps } from "../types";
+
+// prettier-ignore
+export const ${componentName}Icon = forwardRef<SVGSVGElement, IconProps>(
+  ({ color = "currentColor", ...props }, forwardedRef) => {
+    return (
+      ${jsx}
+    );
+  }
+);
+
+${componentName}Icon.displayName = "${componentName}Icon";
+`;
+
+const transformFilename = (filename: string) => {
+  return path.basename(filename, path.extname(filename)) + ".tsx";
+};
+
+export const config = {
+  inputDir: "./icons",
+  outputDir: "./src/gen",
+  template,
+  transformFilename,
+  svgProps: {
+    width: "16",
+    height: "16",
+    fill: "{color}",
```

Comment:
Something to keep in mind: in the current icons, color isn't always in `fill`, sometimes it's `stroke={color}`. In case we'll move existing icons to this or something...

Reply:
We'll figure it out